File: Classes/Extensions/InflectorKit+Helper.m
//
//  InflectorKit+Helper.m
//  iSub
//
//  Created by Benjamin Baron on 1/14/21.
//  Copyright © 2021 Ben Baron. All rights reserved.
//

#import "InflectorKit+Helper.h"
#import <InflectorKit.h>

@implementation NSString(Helper)

- (NSString *)pluralize:(NSInteger)count {
    return count == 1 ? self : self.pluralizedString;
}

@end

File: Classes/Extensions/InflectorKit+Helper.swift
//
//  InflectorKit+Helper.swift
//  iSub
//
//  Created by Benjamin Baron on 1/18/21.
//  Copyright © 2021 Ben Baron. All rights reserved.
//

import InflectorKit

import Foundation

extension String {
    func pluralize(amount: Int) -> String {
        return amount == 1 ? self : self.pluralized
    }
}


